test/sql/test_jit/R/test_arithmetic

```diff
@@ -0,0 +1,167 @@
+-- name: test_arithmetic
+DROP TABLE IF EXISTS `jit_basic`;
+-- result:
+-- !result
+CREATE TABLE IF NOT EXISTS `jit_basic` (
+	k1 BIGINT, 
+	k2 BIGINT, 
+	v1 BIGINT REPLACE, 
+	v2 BIGINT REPLACE, 
+	v3 BIGINT REPLACE,
+	v4 BIGINT REPLACE, 
+	v5 BIGINT REPLACE, 
+	v6 BIGINT REPLACE,
+	v7 BIGINT REPLACE,
+	v8 BIGINT REPLACE,
+	v9 BIGINT REPLACE
+) 
+ENGINE = olap 
+AGGREGATE KEY(k1, k2) 
+DISTRIBUTED BY HASH(k1) BUCKETS 10 
+PROPERTIES( 
+	"replication_num" = "1" 
+);
+-- result:
+-- !result
+INSERT INTO jit_basic (k1, k2, v1, v2, v3, v4, v5, v6, v7, v8, v9) VALUES
+(0, 9223372036854775807, -9223372036854775808, -5123, 5123, -2147483648, 2147483647, 0, 0, -32768, 32767),
+(100, -9223372036854775808, 9223372036854775807, -100, 200, -200, 0, 10000, -10000, 5000, -5000),
+(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0),
+(-1000, 1000, 500, -500, 250, -250, 125, -125, 100000, -100000, 50000),
+(100, -200, 300, -400, 500, -600, 700, -800, 900, -1000, 1100),
+(-56789, 56789, 12345, -12345, 67890, -67890, 0, 11111, -11111, 9999, -9999),
+(1, -1, 2, -2, 3, -3, 4, -4, 5, -5, 6),
+(32767, -32768, 2147483647, -2147483648, 9223372036854775807, -9223372036854775808, 100, -100, 200, -200, 0),
+(9999999, -9999999, 8888888, -8888888, 7777777, -7777777, 6666666, -6666666, 5555555, -5555555, 4444444),
+(0, 1, 131, -142, 1234, 134, -12, 17345, 63456, 341, 14),
+(2147483647, -2147483648, 32767, -32768, 100, -100, 200, -200, 0, 1, -1),
+(-1, 1, -2, 2, -3, 3, -4, 4, -5, 5, -6),
+(9223372036854775807, -9223372036854775808, 2147483647, -2147483648, 32767, -32768, 100, -100, 200, -200, 0),
+(1000, -1000, 500, -500, 250, -250, 125, -125, 100000, -100000, 50000),
+(200, -400, 600, -800, 1000, -1200, 1400, -1600, 1800, -2000, 2200),
+(-123456, 123456, 78901, -78901, 234567, -234567, 0, 54321, -54321, 67890, -67890),
+(7, -14, 21, -28, 35, -42, 49, -56, 63, -70, 77),
+(9876543, -9876543, 13579246, -13579246, 2468642, -2468642, 86420964, -86420964, 11111, -11111, 99999),
+(1234, -645, 9223372036854775807, -9223372036854775808, 1, 2, 3, 0, -1, -2, -3)
+(754, -14, 9223372036854775807, -9223372036854775808, 0, 0, 0, 0, 0, 0, 0);
+-- result:
+E: (1064, "Getting syntax error at line 21, column 0. Detail message: Unexpected input '(', the most similar input is {<EOF>, ';'}.")
```

Comment:
??

Reply:
A comma was missing when adding test cases. I will fix it.